examples/contracts/aggregator-consumer/scripts/consumerValidator.ts

```diff
@@ -0,0 +1,72 @@
+import { ethers, starknet, network } from 'hardhat'
+import { Contract, ContractFactory } from 'ethers'
+import { HttpNetworkConfig, StarknetContract, Account } from 'hardhat/types'
+import fs from 'fs'
+import dotenv from 'dotenv'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+
+dotenv.config({ path: __dirname + '/.env' })
+const AGGREGATOR_NAME = 'Mock_Aggregator'
+const UPTIME_FEED_NAME =
+  '../../../../contracts/starknet-artifacts/contracts/cairo/ocr2/SequencerUptimeFeed/sequencer_uptime_feed'
+
+let validator: Contract
+let mockStarkNetMessengerFactory: ContractFactory
+let mockStarkNetMessenger: Contract
+let deployer: SignerWithAddress
+let eoaValidator: SignerWithAddress
+let networkUrl: string
+let account: Account
+
+async function main() {
+  // account = await starknet.getAccountFromAddress(
```

Comment:
is this still used?  We should remove it if not.

Reply:
@qperrot , we uncommented it